<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adjust</class>
 <widget class="QWidget" name="adjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblwidget_adjustItem">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>800</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>400</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btn_adjustAssit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>260</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>辅助+0.00：+2.50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustUnit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MPa A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustMeasure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustAir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通大气</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_adjustView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblwidget_adjustTitle">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>800</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="tblwidget_adjustLock">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>800</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通道 A 校准</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustAuto">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自动调零</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustOk">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>560</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustUnlock">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>解锁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_adjustclose">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>660</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_adjust">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>560</y>
     <width>231</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>单点校准</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>两点校准</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>恢复出厂校准值</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>校准日期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高程修正</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>改变口令</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tblwidget_adjustLock_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/lock.png);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>800</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btn_num_0">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>690</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_Point">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>690</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_7">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>600</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>430</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_9">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>600</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>430</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_8">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>600</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>510</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_clr">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>CE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>430</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_sub">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>600</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_add">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>510</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_10">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>690</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 255, 190);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tblwidget_adjustItem</zorder>
  <zorder>btn_adjustAssit</zorder>
  <zorder>btn_adjustUnit</zorder>
  <zorder>btn_adjustMeasure</zorder>
  <zorder>btn_adjustAir</zorder>
  <zorder>edit_adjustView</zorder>
  <zorder>tblwidget_adjustTitle</zorder>
  <zorder>tblwidget_adjustLock</zorder>
  <zorder>label</zorder>
  <zorder>btn_adjustAuto</zorder>
  <zorder>btn_adjustOk</zorder>
  <zorder>btn_adjustUnlock</zorder>
  <zorder>btn_adjustclose</zorder>
  <zorder>cmb_adjust</zorder>
  <zorder>tblwidget_adjustLock_2</zorder>
  <zorder>btn_num_0</zorder>
  <zorder>btn_num_Point</zorder>
  <zorder>btn_num_7</zorder>
  <zorder>btn_num_1</zorder>
  <zorder>btn_num_9</zorder>
  <zorder>btn_num_3</zorder>
  <zorder>btn_num_4</zorder>
  <zorder>btn_num_8</zorder>
  <zorder>btn_num_6</zorder>
  <zorder>btn_num_clr</zorder>
  <zorder>btn_num_2</zorder>
  <zorder>btn_num_5</zorder>
  <zorder>btn_num_sub</zorder>
  <zorder>btn_num_add</zorder>
  <zorder>btn_num_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
